File: 프로그래머스/unrated/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        
        int count=0;
        int count1=0;
        for(int i=0;i<num_list.length;i++){
           if(num_list[i] % 2==0){
               count++;
           }
           else if(num_list[i] % 2==1){
               count1++;
           } 
        }
        
        answer[0]=count;
        answer[1]=count1;
        
        return answer;
    }
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    
    var count=0;
    var count1=0;
    for(var i=0;i<num_list.length;i++){
        if(num_list[i] % 2==0){
            count++;
        }
        else if(num_list % 2!=0){
            count1++;
        }
    }
    
    answer[0]=count;
    answer[1]=count1;
    
    return answer;
}